app/javascript/controllers/story_controller.js

```diff
@@ -0,0 +1,60 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["content"];
+
+  async connect() {
+    const response = await fetch("/stories/show");
+
+    const data = await response.json();
+
+    const stories = data.stories.map((story) => {
+      return [
+        story.id,
+        story.name.body.replace("<div>", "").replace("</div>", ""),
+        story.day,
+        story.images,
+      ];
+    });
+
+    const cls_stories = stories.map((story) => {
+      return [story[0], story[1], story[2], story[3]];
```

Comment:
```suggestion
      return [...story];
```

You could use the spread operator.